quantum/quantum.c

```diff
@@ -132,6 +132,14 @@ void unregister_code16 (uint16_t code) {
   }
 }
 
+void tap_code16(uint8_t code) {
+  register_code16(code);
+  #if defined(TAP_CODE_DELAY) && TAP_CODE_DELAY > 0
```

Comment:
```suggestion
  #if TAP_CODE_DELAY > 0
```
The `defined(TAP_CODE_DELAY)` isn't required, but it's okay to keep it if you want.